README.md

```diff
@@ -180,6 +187,11 @@ This will ensure that the server executable is up-to-date before running it.
 Second we used the `--unit-testing` flag,
 which will set some testing options, create a clean new database, and load the test courses (inside the `testdata directory).
 
+Additionally, when running the server in `--unit-testing` mode,
```

Comment:
Need to note here (probably in parentheticals) that this is any executable in the `cmd` directory, not just the server.